Assets/_Project/Scripts/UserInterface/Views/MainView.cs

```diff
@@ -0,0 +1,56 @@
+using PolSl.UrbanHealthPath.UserInterface.Components;
+using PolSl.UrbanHealthPath.UserInterface.Initializers;
+using PolSl.UrbanHealthPath.UserInterface.Interfaces;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Views
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class MainView : MonoBehaviour, IInitializable, IPopupable
+    {
+        public RectTransform PopupArea => popupArea;
+
+        [SerializeField] private Button profileButton;
+        [SerializeField] private Button helpButton;
+        [SerializeField] private Button settingsButton;
+        [SerializeField] private Button upperButtonOnMap;
+        [SerializeField] private Button lowerButtonOnMap;
+        [SerializeField] private Button exitButton;
+        [SerializeField] private Header header;
+        [SerializeField] private RectTransform popupArea;
+        [SerializeField] private TextMeshProUGUI lowerButtonText;
+        [SerializeField] private TextMeshProUGUI upperButtonText;
+
+        public void Awake()
```

Comment:
awake doesnt need to be public

Reply:
Not doesn't need to, it should not be public :)

Reply:
Changed in every script.